### PR TITLE
Updated type definitions

### PR DESCRIPTION
**Pull request by netroy**
* make `Provider` & `Notifications` classes extend event-emitter in the type-definitions (just like they do in the code)
* add a type definition for `Notifications#aps`

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -96,7 +122,7 @@ export interface NotificationAlertOptions {
   "launch-image"?: string;
 }
 
-export class Notification {
+export class Notification extends EventEmitter {
```
Comment (florianreinhart) [inline index.d.ts]: `Notification` does not extend `EventEmitter` or am I missing something?

File: index.d.ts [hunk 2]
```diff
@@ -130,6 +156,8 @@ export class Notification {
    * This Object is JSON encoded and sent as the notification payload. When properties have been set on notification.aps (either directly or with convenience setters) these are added to the payload just before it is sent. If payload already contains an aps property it is replaced.
    */
   public payload: any;
+  public aps: Aps;
```
Comment (florianreinhart) [inline index.d.ts]: I am not sure about exposing the `aps` property. If you want to modify the raw payload, you should use the `rawPayload` property. But I am open for discussions.

Comment (ShaharHD) [inline index.d.ts]: @florianreinhart regarding `aps`, it does mention this in [README.md](https://github.com/node-apn/node-apn/blob/master/doc/notification.markdown#notificationpayload)

So it should be exposed, or am I mistaken here?

**Comment by netroy** [comment]
@florianreinhart updated & rebased

**Pull request merged**